src/main/java/frc/robot/RobotMap.java

```diff
@@ -4,10 +4,13 @@ public class RobotMap {
 
     public interface CAN {
 
+        int ALGAE_SPARK = -1;
     }
     
     public interface DIO {
 
+        int TOP_LIMITER = -1;
```

Comment:
`limit` and not `limiter`.
also, top limit of what? its in the robotmap context, not your subsytem's